### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by ridwanediallo**
# Project requirements: 

1️⃣ Now you should improve the look and feel of the application, adding the styles of your choice.
2️⃣ Please keep the general layout of the wireframe, this is the only mandatory requirement.
3️⃣ You can use plain CSS or any CSS framework, it's up to you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
-# Leaderboad
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard:
+
+## Live Demo
```
Comment (FrederickMih) [inline README.md:10]: - [ ] Hi @ridwanediallo, this is not the correct link to your project. Please let's use the proper link here.

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Report an issue
+
+-Go here 👉 [@Link](https://github.com/ridwanediallo/Leaderboad/issues)
```
Comment (FrederickMih) [inline README.md]: - [ ] From the requirements of this project, we are needed to also include the API(s) it consumes.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @ridwanediallo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


# To Highlights 
:arrow_forward: Your repo looks professional :heavy_check_mark: 
:arrow_forward: Followed GitHub flow :heavy_check_mark: 
:arrow_forward: Correctly set up linters :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ridwanediallo) [reply README.md]: > * [ ]  From the requirements of this project, we are needed to also include the API(s) it consumes.

Hello, thank you for the review. However, this request is not in the requirement.
You can check it out here: https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m3_final_touches.md

**Review by prolajumokeoni — APPROVED** [review]
Hi @ridwanediallo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



all-done-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**